site/en/docs/extensions/mv3/user_interface/index.md

```diff
@@ -411,11 +411,12 @@ one or more shortcuts in the manifest under the `"commands"` key.
 
 Commands can be used to provide new or alternative browser shortcuts. The [Tab
 Flipper][sample-tab-flipper] sample extension listens to the
-[`commands.onCommand`][commands-oncommand] event in the [background script][docs-background] and
+[`commands.onCommand`][commands-oncommand] event in the [service worker][docs-background] and
```

Comment:
The [background script] link sort of works because it's a redirect but we should probably rename it and update where it points?